### ISSUE TITLE
Result

### ISSUE THREAD

**Issue by Stubborn-z** [kingsaint/BioMedical-EL#11]
Thank you very much for your contribution to this work. 
I follow your way and the final result is very poor, both pre and f1 are zero points, can you guide me？

**Comment by kingsaint** [comment]
Did you train the model?

**Comment by Stubborn-z** [comment]
Yes. First I run python run_e2e_span.py --data_dir data/BC5CDR/processed_data/ --model_type bert --model_name_or_path biobert --output_dir output --overwrite_output_dir --overwrite_cache --use_hard_and_random_negative --do_train
Then run python run_e2e_span.py --data_dir data/BC5CDR/processed_data/ --model_type bert --model_name_or_path biobert --output_dir output --use_all_candidates --do_eval.
Then put the generated .csv file in the BC5CDR-AllSpan directory, and then run neleval evaluate -g ./BC5CDR-AllSpan/gold.csv ./BC5CDR-AllSpan/pred.csv -m overlap-maxmax::span+kbid - m strong_all_match
But there is something wrong with the result I got, if I am missing something, can you help me look it up？Thank you very much！

**Comment by kingsaint** [comment]
While running the evaluation with `--do_eval`, you need to provide the path to the trained model to `--model_name_or_path` argument, which should have been saved in `--output_dir` at the end of training. Let me know if this helps.

**Comment by Stubborn-z** [comment]
Thank you for your reply！I saved the trained model in the ./out directory according to your method, and then ran python run_e2e_span.py --data_dir data/BC5CDR/processed_data/ --model_type bert --model_name_or_path out --output out -- use_all_candidates --do_eval
Then run neleval evaluate -g ./neleval/BC5CDR-AllSpan/gold.csv ./neleval/BC5CDR-AllSpan/pred.csv -m overlap-maxmax::span+kbid -m strong_all_match. But the results obtained were particularly poor,can you guide me again?

**Comment by atulbunkar** [comment]
Hey, I am also facing same issue as above. I ran the code as explained above and still getting those issues. Can you tell if the above command is correct or not. Thanks.

**Comment by kingsaint** [comment]
@Stubborn-z @atulbunkar  For how many epochs did you run the training? It could be the case that the model is overfitting. Can you try with the `--eval_all_checkpoints` flag along with the `--do_eval` during the evaluation? Also, did you look into the `tensorboard` logs? What does the training loss look like?
